Doc/library/datetime.rst

```diff
@@ -2537,6 +2537,12 @@ Notes:
       string. Otherwise ``%Z`` is replaced by the returned value, which must
       be a string.
 
+      Note that ``%Z`` accepts only the hardcoded values: ``UTC`` and ``GMT``,
```

Comment:
Perhaps we should specify that this is for `strptime`? "Note that `strptime` only accepts certain values for `%Z`: ..."

Reply:
Thanks good suggestion.